.github/workflows/docs.yml

```diff
@@ -1,12 +1,8 @@
 name: Docs
+on: [push]
```

Comment:
But wouldn't this overwrite the  docs whenever a PR is opened? That would mean that the docs are potentially temporarily incorrect

Reply:
Yeah I will fix so it only generates new docs on push to the main branch.

Reply:
👍